sysbuild/Kconfig.lwm2m_carrier

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+menuconfig LWM2M_CARRIER_DIVIDED_DFU
+	bool "Proprietary LwM2M Carrier divided DFU file generation"
+	depends on BOOTLOADER_MCUBOOT
```

Comment:
shouldn't this also depend upon the soc i.e. nrf91?

Reply:
It won't be very useful for other SOCs, I've added SOC_SERIES_NRF91X dependency